internal/cmd/iam/command_groupmapping_list.go

```diff
@@ -0,0 +1,41 @@
+package iam
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *groupMappingCommand) newListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List group mappings.",
+		Args:  cobra.NoArgs,
+		RunE:  c.list,
+	}
+
+	pcmd.AddOutputFlag(cmd)
```

Comment:
Please add `--environment` and `--context`. All Cloud commands have these flags.

Reply:
I took a look at other `iam` resources (non-RBAC and ACL) like `service-account`, `provider`, and `pool`. And none of them have an `--environment` or `--context` flag for their list command. 

I guess the `--context` flag makes sense here for `iam` resources because I could list out these resources for a different platform without having to explicitly change contexts first, but I don't think these issues are env-level, so why would I need an `--environment` flag?

Reply:
Oh, are these resources all at the organization level?

Reply:
Yeah the ones mentioned above and `group-mapping` are org level according to their api-specs: https://github.com/confluentinc/api/blob/master/sso/minispec.yaml#L42
https://github.com/confluentinc/api/blob/master/identity-provider/minispec.yaml#L42
https://github.com/confluentinc/api/blob/master/iam/minispec.yaml#L86